TaskModules/powershell/TaskModuleSqlUtility/SqlPackageOnTargetMachines.ps1

```diff
@@ -477,8 +467,34 @@ function Execute-DacpacDeployment
     $sqlPackageArguments = Get-SqlPackageCmdArgs -dacpacFile $dacpacFile -targetMethod $targetMethod -serverName $serverName -databaseName $databaseName -authscheme $authscheme -sqlServerCredentials $sqlServerCredentials -connectionString $connectionString -publishProfile $publishProfile -additionalArguments $additionalArguments
     Write-Verbose -Verbose $sqlPackageArguments
     
-    $command = "`"$sqlPackage`" $sqlPackageArguments"
-    Write-Verbose "Executing command: $command"
-    RunCommand -command $command -failOnErr $true
+    Write-Verbose "Executing command: $sqlPackage $sqlPackageArguments"
+    Invoke-Command -FileName "$sqlPackage"  -Arguments $sqlPackageArguments
```

Comment:
Let's not override build in function invoke-command instead name it as execute-sqlScript